### PR TITLE
update maintainers

### PR DESCRIPTION
**Pull request by ilyam8**
An attempt to keep the list up to date

### CODE REVIEW

File: charts/netdata/Chart.yaml [hunk 1]
```diff
@@ -14,15 +14,11 @@ sources:
...
-  - name: Karuppiah Natarajan
-    email: karuppiah7890@gmail.com
-  - name: Oleksii Kravchenko
-    email: kamikaze.ua@gmail.com
+  - name: Mateusz Bularz
+    email: mat@netdata.cloud
+  - name: Bill Kalaitzis
+    email: bill@netdata.cloud
+  - name: Witold Duranek
+    email: witold@netdata.cloud
```
Comment (ilyam8) [inline charts/netdata/Chart.yaml:22]: Probably a better way to put a team here, not individual people.

Comment (Ferroin) [reply charts/netdata/Chart.yaml:22]: Agreed. Perhaps we should create an internal alias specifically for those maintaining the Helm chart?

Comment (ilyam8) [reply charts/netdata/Chart.yaml:22]: I was thinking of just using the Cloud SRE team. I don't think that is so important we need a dedicated alias just for helmchart maintainers.

Comment (Ferroin) [reply charts/netdata/Chart.yaml:22]: Well yes, but it’s not _just_ the Cloud SRE team.

Comment (ilyam8) [reply charts/netdata/Chart.yaml:22]: Ok, maintainers is [optional](https://helm.sh/docs/topics/charts/#the-chartyaml-file). I think we can just remove the field. I think it is ok, maintained by Netdata.

Comment (ilyam8) [reply charts/netdata/Chart.yaml]: @witalisoft added Cloud SRE team, please approve

**Review by BillKalaitzis — APPROVED** [review]

**Review by underhood — APPROVED** [review]

**Pull request merged**